packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -257,40 +317,59 @@ declare module 'lightning/uiRecordApi' {
         updateable: boolean;
     }
 
-    interface ThemeInfoRepresentation {
+    /**
+     * Theme info.
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface ThemeInfoRepresentation {
         color: string;
-        iconUrl: string;
+        iconUrl: string | null;
     }
 
+    /**
+     * Object metadata.
+     *
```

Comment:
https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/ui_api_responses_object_info.htm

Reply:
The UI API doc doesn't seem to have some of the properties listed here. 
@jbleyleSF, I'm not sure if all the properties here should also be in the UI API doc. Any ideas?